lisa/features/gpu.py

```diff
@@ -3,45 +3,61 @@
 
 import re
 from enum import Enum
-from typing import Any
+from typing import Any, List, Set
 
-from lisa.base_tools.wget import Wget
+from lisa.base_tools import Uname, Wget
 from lisa.feature import Feature
 from lisa.operating_system import Redhat, Ubuntu
-from lisa.tools import Uname
+from lisa.sut_orchestrator.azure.tools import LisDriver
+from lisa.tools import Lspci, Lsvmbus
 from lisa.util import LisaException, SkippedException
 from lisa.util.logger import get_logger
 
 FEATURE_NAME_GPU = "Gpu"
 
-ComputeSDK = Enum(
-    "ComputeSDK",
-    [
-        # GRID Driver
-        "GRID",
-        # CUDA Driver
-        "CUDA",
-    ],
-)
-
 # Link to the latest GRID driver
 # The DIR link is
 # https://download.microsoft.com/download/9/5/c/95c667ff-ab95-4c56-89e0-e13e9a76782d/NVIDIA-Linux-x86_64-460.32.03-grid-azure.run
 DEFAULT_GRID_DRIVER_URL = "https://go.microsoft.com/fwlink/?linkid=874272"
 
 
 class Gpu(Feature):
+
+    ComputeSDK = Enum(
```

Comment:
why move it in class? If moving it into class, the name conversion is different, it should be like `compute_sdk`. But first question is, why move it in?

Reply:
moved it to class since it won't be used anywhere without this class looks?

Reply:
The Enum usually is considered as the same level as class. So it's not common to put a Enum in a class. But it's ok, I don't have strong preference. The name should be like `compute_sdk`, if it's in a class.

Reply:
Ok sounds right. I moved it out of the Gpu class.